emails/utils.py

```diff
@@ -202,17 +203,29 @@ def get_welcome_email(request: HttpRequest, user: User, format: str) -> str:
 
 @time_if_enabled("ses_send_raw_email")
 def ses_send_raw_email(
+    *,
```

Comment:
This syntax means that all the following parameters are keyword-only, allowing me to mix parameters with and without defaults